app/controllers/users_controller.rb

```diff
@@ -8,21 +7,20 @@ class UsersController < ApplicationController
 
 
   def index
-    @users = @users.featured.paginate :per_page => 12,
-                                      :page => params[:page]
+    @users = @users.featured.paginate per_page: 12, page: params[:page]
   end
 
   def rookies
-    @users = User.rookies.paginate :per_page => 12,
-                                   :page => params[:page]
+    @users = User.rookies.paginate per_page: 12, page: params[:page]
     render :index
   end
 
   def show
-    @recipes = @user.recipes.user_page(params[:order]).
-      paginate :page => params[:page], :per_page => 10
+    @recipes = @user.recipes.
+      user_page(params[:order]).
+      paginate page: params[:page], per_page: 10
     @categories = @user.categories.order(:name)
-    render :layout => 'users'
+    render layout: 'users'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.